x/mongo/driver/operation/find_and_modify.toml

```diff
@@ -60,6 +60,11 @@ type = "array"
 minWireVersionRequired = 6
 documentation = "ArrayFilters specifies an array of filter documents that determines which array elements to modify for an update operation on an array field."
 
+[request.hint]
+type = "value"
+minWireVersionRequired = 8
+documentation = "Hint specifies the index to use. This option is only valid for server versions >= 4.4. Server versions between 4.2 and 4.3, inclusive, will error if this option is set. For server versions < 4.2, the driver will error if this option is set."
```

Comment:
No need to reference dev releases of the server.  This can just say 

    Server 4.2 will error if this option is set.

Reply:
Done.